test/remote/gateways/remote_trans_first_test.rb

```diff
@@ -70,12 +70,12 @@ def test_failed_purchase
     assert_equal 'Insufficient funds', response.message
   end
 
-  def test_successful_refund
```

Comment:
Okay so keep the refund method but maybe test for failure with the right message?